Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -44,7 +44,7 @@ namespace Microsoft.PythonTools.Profiling {
     // This attribute tells the PkgDef creation utility (CreatePkgDef.exe) that this class is
     // a package.
     [PackageRegistration(UseManagedResourcesOnly = true)]
-    [Description("Python Tools Profiling Package")]
+    [Description("Python Tools Profiling Package")] // TODO: Localization: does this do anything?
```

Comment:
Do you know what this is for?  RTVS and Cookiecutter package class don't have it.  I'm hoping we can just remove it.  Most of PTVS packages have the attribute.